libs/langchain/langchain/document_loaders/pubmed.py

```diff
@@ -0,0 +1,29 @@
+from typing import Iterator, List, Optional
+
+from langchain.docstore.document import Document
+from langchain.document_loaders.base import BaseLoader
+from langchain.utilities.pubmed import PubMedAPIWrapper
+
+
+class PubMedLoader(BaseLoader):
+    """Loads a query result from PubMed biomedical library into a list of Documents."""
+
+    def __init__(
+        self,
+        query: str,
+        load_max_docs: Optional[int] = 3,
+    ):
+        self.query = query
+        """The query to be passed to the PubMed API."""
```

Comment:
nit: I generally follow: https://google.github.io/styleguide/pyguide.html#384-classes

Do you know if the attribute comments will be pulled out by sphinx to document the class attributes? (if not documented at the class level?)

As a user I find it helpful when __init__ is documented since it appears in tab-completion in jupyter / ipython

Reply:
Thanks for the link!